docs/conf.py

```diff
@@ -0,0 +1,53 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+import sys, os, datetime
+
+sys.path.append(os.path.abspath('_extensions'))
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = 'Godot OpenXR Vendors plugin'
+copyright = (
+    "2014-present Juan Linietsky, Ariel Manzur and the Godot community (CC BY 3.0)"
+)
+author = "Juan Linietsky, Ariel Manzur and the Godot community"
```

Comment:
The copyright and author should be updated to match the ones used for this repo.